src/framework/print-export/PESketchMethods.m

```diff
@@ -79,9 +79,17 @@ + (void)displayFlashMessage:(NSString *)message document:(MSDocument *)document
 
 + (MSImmutableLayerGroup *)detachedLayerGroupRecursively:(BOOL)recursively withDocument:(MSImmutableDocumentData *)documentData symbolInstance:(MSImmutableSymbolInstance *)symbolInstance {
     SEL selector = NSSelectorFromString(@"detachedLayerGroupRecursively:withDocument:visitedSymbols:");
-    typedef id (*MethodType)(MSImmutableSymbolInstance *, SEL, BOOL, MSImmutableDocumentData *, id);
-    MethodType method = (MethodType)[symbolInstance methodForSelector:selector];
-    return method(symbolInstance, selector, recursively, documentData, nil);
+    // This only exists in Sketch 55 and earlier
+    if ([symbolInstance respondsToSelector:selector]) {
+        typedef id (*MethodType)(MSImmutableSymbolInstance *, SEL, BOOL, MSImmutableDocumentData *, NSSet *);
+        MethodType method = (MethodType)[symbolInstance methodForSelector:selector];
+        return method(symbolInstance, selector, recursively, documentData, nil);
+    } else {
```

Comment:
```suggestion
    }
```

Since you return if the condition is met you can drop the `else` block and keep the default logic one indentation level more to the left.